ads/lockerdome.md

```diff
@@ -0,0 +1,35 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# LockerDome
+
+[LockerDome](https://lockerdome.com)
+
+## Example
+
+```html
+<amp-ad width="300" height="250"
+    type="lockerdome"
+    data-slot="10238273616545382">
+</amp-ad>
+```
+
+## Configuration
+
+Supported parameters:
+- width
```

Comment:
There's no need to specify `width` and `height` here, as they are supported for all `amp-ad`, not some special ad type.